advanced-configuration/pod-overlay/README.md

```diff
@@ -0,0 +1,91 @@
+
+Confluent for Kubernetes (CFK) pod overlay feature allows you to use Kubernetes pod functionality that's not currently 
+supported in CFK API(`<component>.spec.podTemplate` section).
+Customer can leverage pod overlay feature in Zookeeper, Kafka, Connect, Schema Registry, Kafka Rest Proxy and 
+Control Center by adding an annotation(`platform.confluent.io/pod-overlay-configmap-name: <cm-name>`) which point to 
+a configmap that contains pod overlay configuration in CR. There are fields not allowed to be overwritten by pod overlay, those fields are critical to the functionality of application pod.  
+The valid pod overlay configuration will be [strategically merged](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/#use-a-strategic-merge-patch-to-update-a-deployment) with 
+pod spec inside statefulSet generated by CFK to form the final pod configuration for the application.
+
+In this example, you'll set up a Confluent Platform(Zookeeper and Kafka), and configure pod overlay for Zookeeper and Kafka respectively. To configure pod overlay, you'll first create a configmap resource containing the pod overlay yaml content, and later, annotate CR.
+
+## Set the current tutorial directory
+
+Set the tutorial directory under the directory you downloaded this Github repo:
+
+```   
+export TUTORIAL_HOME=<Github repo directory>/advanced-configuration/pod-overlay
+```
+
+## Deploy Confluent for Kubernetes
+
+This workflow scenario assumes you are using the namespace `confluent`.
+
+Set up the Helm Chart:
+
+```
+helm repo add confluentinc https://packages.confluent.io/helm
+```
+
+Install Confluent For Kubernetes using Helm:
+
+```
+helm upgrade --install operator confluentinc/confluent-for-kubernetes -n confluent
+```
+
+Check that the Confluent For Kubernetes pod comes up and is running:
+
+```
+kubectl get pods
+```
+
+## Deploy Confluent Platform
+
+Deploy Confluent Platform:
+
+```
+kubectl apply -f $TUTORIAL_HOME/confluent-platform.yaml
+```
+
+Check that all Confluent Platform resources are deployed:
+
+```   
+kubectl get confluent -n confluent
+```
+
+## Configure pod overlay
+- Create configmap used by pod overlay (required data: `key: pod-template.yaml`, `value: <yaml content>` which conform with kubernetes [StatefulSetSpec](https://github.com/kubernetes/api/blob/master/apps/v1beta1/types.go#L208) API). 
+yaml content must begin with `spec`(refer to `statefulset.spec`). Users are only allowed to configure fields inside `statefulset.spec.template`. If they try to set fields outside of it, the configuration will be considered as invalid.
+
+  - option 1: add a custom init container
+  
+    `kubectl -n confluent create configmap foo-pod-overlay --from-file=pod-template.yaml=$TUTORIAL_HOME/extra-init-container.yaml`
+  - option 2: enable QoS class of BestEffort
+  
+    `kubectl -n confluent create configmap foo-pod-overlay --from-file=pod-template.yaml=$TUTORIAL_HOME/enable-best-effort-qos.yaml`
```

Comment:
these look good, curious what are the top use cases again

Reply:
these two use cases have associated FF tickets so I figured these would be top use cases and provided example here